Wesley0010012/PasswordGeneratorApi
# Feat/save password controller

Description
This pull request have the objective of implements the SavePasswordController.

How to Test
To test this new component, it will need to use phpunit package, one of default dependencies of Laravel project.

Expected Result
Expects to can make SavePassword actions.
